Cargo.toml

```
@@ -9,8 +9,9 @@ authors = ["Gear Technologies"]
 gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"], branch = "stable" }
 auction-io = { path = "io" }
 primitive-types = { version = "0.12.1", default-features = false, features = ["scale-info"] }
-nft-io = { git = "https://github.com/gear-dapps/non-fungible-token.git", tag = "0.2.5" }
-gear-lib = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.3" }
+nft-io = { git = "https://github.com/gear-dapps/non-fungible-token.git", branch = "new_meta" }
+gear-lib = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.4" }
+gmeta = { git = "https://github.com/gear-tech/gear", features = ["codegen"] }
```

```suggestion
gmeta = { git = "https://github.com/gear-tech/gear" }
```
Makes sense only in the `state`crate.